qualtrix/settings.py

```diff
@@ -32,11 +49,33 @@
         API_TOKEN = config["api_token"]
         BASE_URL = config["base_url"]
         DIRECTORY_ID = config["directory_id"]
+        LIBRARY_ID = config["library_id"]
+        REMINDER_MESSAGE_ID = config["reminder_message_id"]
+        INVITE_MESSAGE_ID = config["invite_message_id"]
+        MAILING_LIST_ID = config["mailing_list_id"]
+        FROM_EMAIL = config["from_email"]
+        REPLY_TO_EMAIL = config["reply_to_email"]
+        FROM_NAME = config["from_name"]
+        INVITE_SUBJECT = config["invite_subject"]
+        REMINDER_SUBJECT = config["reminder_subject"]
+        SURVEY_LINK_TYPE = config["survey_link_type"]
+        DEMOGRAPHICS_SURVEY_LABEL = config["demographics_survey_label"]
```

Comment:
Do we want to create a more structured config? all of these variables at global scope in the config seems fine to me

Reply:
ok